src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -100,6 +100,11 @@ Status MediapipeGraphDefinition::dryInitializeTest() {
 Status MediapipeGraphDefinition::validate(ModelManager& manager) {
     SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Started validation of mediapipe: {}", getName());
     ValidationResultNotifier notifier(this->status, this->loadedNotify);
+    auto& models = manager.getModels();
+    if (std::find_if(models.begin(), models.end(), [this](auto pair) { return this->getName() == pair.first; }) != models.end()) {
+        SPDLOG_LOGGER_ERROR(modelmanager_logger, "Mediapipe graph name: {} is already occupied by model or pipeline.", this->getName());
+        return StatusCode::MEDIAPIPE_GRAPH_INITIALIZATION_ERROR;
```

Comment:
Have separate status code for this error as we won';t be able to differentiate between MP error during graph.Initialize(), and this configuration issue.